polyfills/Symbol/toStringTag/config.toml

```
@@ -8,12 +8,12 @@ docs = "https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Ob
 android = "*"
 bb = "10 - *"
 chrome = "<49"
-edge = "<13"
-edge_mob = "<13"
+edge = "<15"
+edge_mob = "<15"
 firefox = "<51"
 ie = "*"
 ie_mob = "*"
 opera = "<37"
```

```suggestion
opera = "<37"
op_mob = "<37"
```

Should this be `<36`?

---

👍